webapp/tests/Unit/Controller/API/BaseTestCase.php

```diff
@@ -60,6 +60,20 @@ protected function setUp(): void
             ->findOneBy(['shortname' => 'demo']);
     }
 
+    /**
+     * Get the contest ID of the demo contest based on the data source setting.
+     *
+     * @return string
+     */
+    protected function getDemoContestId(): string
```

Comment:
Is this not something which we also want to reuse in non API tests?

Reply:
If we do we move it (and the property that backs it) to the normal base test case. Now this method was here but the property was in the non-API class, which PHPStan disliked (and so did I).